### DISCUSSION TITLE
Single source of truth for boards that we support and usecases.

### DISCUSSION THREAD

**Discussion by 0Grit** [golioth/golioth-zephyr-sdk#126] [Questions]
@mniestroj 
1. Can we create a single source of truth for our 'flagship' boards
2. Can we run CI on those boards based on their device tree?

_Originally posted by @0Grit in https://github.com/golioth/zephyr-sdk/pull/118#pullrequestreview-823298786_

**Comment by mniestroj** [comment]
>Can we create a single source of truth for our 'flagship' boards

Nothing comes to my mind other than creating a list in README.

>Can we run CI on those boards based on their device tree?

Could you elaborate what you have in mind here? It would be great to run CI (specifically meaning runtime testing) on those boards. What do you mean by "based on their device tree"?

**Reply by 0Grit** [reply]
1. Create a list of boards which we actively support and validate
2. Use that list in CI to run builds for all boards which have the pre-requisites to be able to run a particular sample.

**Comment by vitprajzler** [comment]
Since this is an engineering-wide topic, where supported boards might come from various sources, please consider that as beyond scope of just Zephyr. 

The device catalog in the docs is a good option as a starting point for that single source of truth. It can be moved out of the docs into a separate repository. 

I'd suggest @0Grit you scope out the needs for what the single source of truth will need to integrate into / how it's going to be used, and then decide on the "how".
